### DISCUSSION TITLE
想一个服务器搭建多个百阅发卡  怎么操作呢

### DISCUSSION THREAD

**Discussion by etong33** [Baiyuetribe/kamiFaka#31] [Bug反馈]
如题
我是宝塔面板 第一个成功搭建，还想搭建一个 该如何操作呢？

**Comment by Baiyuetribe** [comment]
默认的8000端口、本地保存的数据或数据库改变下，就可以。具体的请阅读docker相关的基础教程

**Comment by huangfengye** [comment]
你docker的端口改改映射不就完事了。。。内部端口都是8000，外部端口你可以是8001 8002以此类推
